reference-implementation/test/readable-byte-stream.js

```diff
@@ -1,25 +1,13 @@
 const test = require('tape-catch');
 
 test('ReadableByteStream can be constructed with no errors', t => {
```

Comment:
All the test titles in this file will need updating to not be "ReadableByteStream" but I am not sure what the right substitute is. Maybe "ReadableStream with pullInto".